tools/ci_build/github/android/build_aar_package.py

```diff
@@ -99,6 +101,22 @@ def _build_aar(args):
         abi_build_dir = os.path.join(intermediates_dir, abi)
         abi_build_command = [*base_build_command, "--android_abi=" + abi, "--build_dir=" + abi_build_dir]
 
+        if qnn_android_build:
```

Comment:
this block probably can go outside of the ABI build loop. maybe just append `--qnn_home` to `base_build_command` if applicable.